datahub-web-react/src/app/search/AdvancedSearchFilterConditionSelect.tsx

```diff
@@ -0,0 +1,93 @@
+import { Select } from 'antd';
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import { FacetFilterInput } from '../../types.generated';
+import { ANTD_GRAY } from '../entity/shared/constants';
+import { FIELDS_WHO_USE_CONTAINS_OPERATOR } from './utils/constants';
+
+type Props = {
+    filter: FacetFilterInput;
+    onUpdate: (newValue: FacetFilterInput) => void;
+};
+
+const { Option } = Select;
+
+// We track which fields are collection fields for the purpose of printing the conditions
+// in a more gramatically correct way. On the backend they are handled the same.
+const filtersOnNonCollectionFields = [
+    'description',
+    'fieldDescriptions',
+    'fieldPaths',
+    'removed',
+    'typeNames',
+    'entity',
+    'domains',
+    'origin',
+];
+
+function getLabelsForField(field: string) {
+    if (FIELDS_WHO_USE_CONTAINS_OPERATOR.indexOf(field) >= 0) {
```

Comment:
can we just use `.includes()` here instead of `indexOf`? I think it makes the code a lot more confusing

Reply:
includes has worse browser support: https://caniuse.com/?search=includes